modules/form/package.json

```
@@ -0,0 +1,51 @@
+{
+    "name": "@wso2is/form",
+    "version": "1.0.0",
+    "description": "Form library for WSO2 Identity Server front end applications.",
+    "keywords": [
+        "WSO2",
+        "WSO2 IS",
+        "WSO2 Identity Server",
+        "WSO2 Identity Apps",
+        "form"
+    ],
+    "main": "dist/src/index.js",
+    "types": "dist/typings/index.d.ts",
+    "scripts": {
+        "prebuild": "npm run clean",
+        "build": "npm run compile",
+        "clean": "rimraf ./dist",
+        "compile": "npx run-script-os",
+        "compile:win32": "..\\..\\node_modules\\.bin\\tsc -p tsconfig.json",
+        "compile:default": "../../node_modules/.bin/tsc -p tsconfig.json",
+        "lint": "tslint -t stylish --project .",
```

Let's use eslint here.
```suggestion
        "lint": "eslint --ext .js,.jsx,.ts,.tsx .",
```